hedera-node/configuration/testnet/bootstrap.properties

```
@@ -1,5 +1,18 @@
+# This file needs to exist even though there aren't any properties in it.
+# Downstream processes depend on its existence.
+# This is an exact replica of file 121 on testnet and must match application.properties exactly.
+# It will be used by mono-service for all the property overrides.
+# This file will be deprecated when we deprecate using the mono-service config and fully move to hedera-config.
+accounts.maxNumber=20_000_000
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+staking.rewardRate=48630136986000
+tokens.nfts.maxAllowedMints=10_000_000
+#Overrides that differ based on the network
 ledger.id=0x01
 contracts.chainId=296
 bootstrap.genesisPublicKey=e06b22e0966108fa5d63fc6ae53f9824319b891cd4d6050dbf2b242be7e13344
 accounts.blocklist.enabled=false
-accounts.blocklist.path=
+accounts.blocklist.path=
```

There needs to be a newline at the end of the file for git to be happy.

```suggestion
accounts.blocklist.path=

```

---

yes! Did on all files. Thanks